src/components/VBanner/VBanners.vue

```diff
@@ -37,12 +37,17 @@ export default defineComponent({
     const translationBannerId = computed<TranslationBannerId>(
       () => uiStore.translationBannerId
     )
+
+    const dismissBanner = (bannerKey: TranslationBannerId) => {
+      uiStore.dismissBanner(bannerKey)
+    }
+
     return {
       translationBannerId,
       shouldShowMigrationBanner,
       shouldShowTranslationBanner,
 
-      dismissBanner: uiStore.dismissBanner,
+      dismissBanner,
```

Comment:
Did this change have a noticeable effect on the banner behavior? It seems like a welcome change regardless.